brat/Brat/Error.hs

```diff
@@ -181,7 +181,7 @@ instance Show SrcErr where
   show (SrcErr prelim Err{msg=msg}) = unlines [prelim, "  " ++ show msg]
 
 errHeader :: String -> String
-errHeader name = "Error in " ++ name ++ ":"
+errHeader name = "Error in " ++ name
```

Comment:
Reinstate the `:` here, it's disappeared from messages that don't have a line number